### DISCUSSION TITLE
Dashboard

### DISCUSSION THREAD

**Discussion by carlosvirreira** [Shelf-nu/shelf.nu#438] [Ideas]
Checking dashboard design with our users. Asking for input or comments. 

We got the following from one of our users:

Preview shown: 

<img width="731" alt="Screenshot 2023-10-06 at 13 17 12" src="https://github.com/Shelf-nu/shelf.nu/assets/12449255/1d8834c6-5e93-430d-b034-174878a90777">

I like the dashboard! Very crisp and clear. I suppose for functionality it might be good to **drill through the visualisations on Category** for example to **summarise how many are tagged as “Computer”** for example, out of all of the assets which are considered **“Office Equipment”.**
 
I use this as an example as, just to give you some background, I will be predominantly involved in managing our IT assets using Shelf.nu which would be categorised under **Office Equipment**. We’re going to be using the tags to further categorise these assets and allow us to search.
 
**Another potential thing I can think of for inclusion on the dashboard is highlighting whether any assets are in a state requiring action such as maintenance/servicing/repair.**
 
Many of our assets on the **Operations side are working tools and testing equipment** (which undergo an annual calibration) for example.
 
We’re managing this aspect of it separately at the moment but it is perhaps something which your other users would find useful if it was capable of flagging issues like this.

**Comment by carlosvirreira** [comment]
@jurrejansen lets use this reply as input for a potential iteration/improvement

**Comment by hunar1312** [comment]
I believe this to be the best enhancement in shelf, having a dashboard with all the analysis and activities the user has performed in shelf as this will power the user with better management and tracking of their assets specially for the users who deal with `n` number of assets daily and an overall analysis of the activities with their assets could ease their analyzation in tracking assets and this is the motto of shelf.

Good to have these additions in dashboard page
- having the stats of each `relationship with asset module` like location, custodies, etc
- a duration selector to have the analysis of the activity in shelf over a particular period of time
- assets registered in the selected duration
- assets removed from the system in the selected duration

**Reply by carlosvirreira** [reply]
Super good suggestions and rationale, @hunar1312 I will discuss this with @jurrejansen and we will give a reaction and update the conversation!

**Comment by carlosvirreira** [comment]
Congrats on the dashboard release guys! We can launch some small improvements to it (calc of asset value) < we ran this through our advisor and he really likes that we track this globally. Let's go!
